huaweicloud/services/acceptance/as/resource_huaweicloud_as_policy_test.go

```diff
@@ -39,7 +39,7 @@ func TestAccASPolicy_basic(t *testing.T) {
 			{
 				Config: testASPolicy_update(rName),
 				Check: resource.ComposeTestCheckFunc(
-					resource.TestCheckResourceAttr(resourceName, "status", "INSERVICE"),
```

Comment:
Please check the total action operations:
step1: action = pause; step2: action = resume; step3: action = pause